electron/main.js

```diff
@@ -14,24 +15,14 @@ function createWindow() {
     },
   })
 
-  if (process.env.DEV_BUILD) {
-    // Enable loadFile: showing images from external folder
-    // - disables using localhost:3300 & React hot-reload
-    // - requires running `yarn run build`
-    mainWindow.loadFile('build/index.html')
```

Comment:
Is this now removed? Or moved? Or deleted by accident?

Reply:
Okay... If you said you tested this with your own content, then it must work! Have to try out myself too! :)

Realised also that the pathname had changed a bit:
```
// New
pathname: path.join(__dirname, '../build/index.html'),
// Old
pathname: path.join(__dirname, '../index.html'),
```